core/frontend/src/BriefcaseConnection.ts

```diff
@@ -146,13 +146,68 @@ class ModelChangeMonitor {
   }
 }
 
+/** Default settings that can be used to control the behavior of [[Tool]]s that modify a [[BriefcaseConnection]].
```

Comment:
i'd remove the word "Default". It implies there are other settings somewhere that override. You say they can be ignored.

Reply:
Done; also renamed `BriefcaseConnection.activeSettings` to `editorToolSettings.